### PR TITLE
Reworked exterior engine sounds, new flap sounds, and other fixes

### PR DESCRIPTION
**Pull request by hotshotp**
## Summary of Changes

Hello, this PR will have a lot of long awaited changes to be made! Starting off with the small things:

-I have removed the 'FMGCtest' wwise event (triple click or crc + triple click) for @lukecologne and @beheh to use on their FWC implementation
-I have added new autopilot disconnect wwise events: the cavalry charge (the single beebeep sound) is "cavcharge" and the triple click is "3click"

Now onto big changes:

-Added cabin fan sound, will play in the cabin when power is on and the cab fans button on the OVHD panel is pressed
-Reworked flap sounds again, hopefully for the last time
-Totally reworked the outside engine sounds, made it so there is a better idle sound and a much more satisfying rumble behind the engine at high thrust levels
-Added a completely new idle sound again, has more realistic bass now
-On interior startup, I made the transition from startup to idle a little snappier to match IRL 
-Added new APU start contactor sounds. When flap is open, two clunks will sound. When N1 reaches 55%, the two clunks will play but will be more spaced out (disengagement of the contactors)
-Added new nose wheel liftoff sound which sounds more like a vibration that just loud shaking
-TOGA pitch is now a bit higher, as before toga was not high enough and now it is like the real leap

This is it for now, I feel like I am completely done with LEAP sounds once this is merged. I will begin working on IAE after this.

## References

https://www.youtube.com/watch?v=E6SB49LHnzI This video I used to get the TOGA pitch
https://www.youtube.com/watch?v=i3y6F2zVD5Q for startup
https://www.youtube.com/watch?v=qlQc9LyeGUo for apu start contactors (1:03 for the sound)
https://www.youtube.com/watch?v=smlgMd2s0tU for idle sound

## Additional context

Please test and tell me on discord if there are any issues that you find. I want to be sure now and finish up this engine so I can start working on IAE V2500 sounds.

Discord username (if different from GitHub):

Boris#9134

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by tjburni** [comment]
Category: QA Tester
Name: tjburni #0409
Date of testing: Current date 26.11.2020
Version of the sim: 1.11.6.0
PR Tested: #2106
Object of test:Reworked exterior engine sounds, new flap sounds, and other fixes

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. loaded in the sim 
2. started the plane 
3. lookes at engine flaps and overall sounds( all in pr described changes)
4.
5.

Flight notes (If required): tested at different engine speeds and tested flaps sounds
idle sounds much better now

Issues (If Any): none
Overall Rating: Good
Conclusions of the testing: sounds are better than before and the plane overall sounds more realistic

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 1]
```diff
@@ -863,6 +923,10 @@
         <Sound WwiseData="true" WwiseEvent="aural_100above" NodeName="WIPER_BASE_L" />
         <Sound WwiseData="true" WwiseEvent="aural_retard" />
         <Sound WwiseData="true" WwiseEvent="aural_v1" />
+        <Sound WwiseEvent="FMGCtest" WwiseData="true" NodeName="PEDALS_LEFT" Continuous="false" />
```
Comment (beheh) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:926]: Is this supposed to be here?

Comment (hotshotp) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:926]: Yes, that just defined that the event will have the node in the cockpit. It won't trigger unless you ask it to

**Comment by mikecbr** [comment]
@hotshotp  what about the sound in cockpit on cruise ? you shouldnt hera the engines at all.....

**Comment by hotshotp** [comment]
> @hotshotp what about the sound in cockpit on cruise ? you shouldnt hera the engines at all.....

that will be done later in another pr

**Comment by MapleNAV-TCASFAULT** [comment]
Category: QA Tester
Name: ChristopherXie#3904
Date of testing: Current date 11/26/2020
Version of the sim: 11.6.0
PR Tested: #2106 
Object of test: aircraft sounds

Testing technique: Entered the sim to test the PR
Steps you've made to test the Pull Request:
1. Started cold and dark, listened for the APU contact sounds
2. Started the engines and looked over the wings to hear new flap sounds
3. Did a takeoff, listened to the new TOGA sounds and the new rotation sounds, also exterior sounds
4. Did a takeoff on spawning on the runway
5. Did a landing by slewing to a position.

Flight notes (If required):

Issues (If Any): 
Overall Rating: Good
Conclusions of the testing: Great sounds, love the new TOGA sounds, and all the minor changes. IMO the TOGA sound pitch does seem to be a bit too high, but after referencing a few videos, it does sound very similar. Something just seems off. Otherwwwise, fantastic job, can't wait for the next PR! :)

**Comment by AviationSpotter** [comment]
Category: QA Tester
Name: Aviation Spotter#9102
Date of testing: 11/26/2020
Version of the sim: 1.11.6.0
PR Tested: #2106
Object of test: Added and tweaked aircraft sounds

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1. Spawned Cold & Dark at a gate
2. Performed normal operations to takeoff
3. Did a circuit at KJFK
4. After landing went to a gate to shutdown

Issues (If Any): None
Overall Rating: Good
Conclusions of the testing: Engine and flaps sounds are in fact improved sounding more like the real aircraft. SGTM

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 2]
```diff
@@ -197,52 +215,50 @@
...
 
-        <Sound WwiseEvent="ISpoolHighR" WwiseData="true" NodeName="Engine_PIVOT_RIGHT_2" SimVar="TURB ENG N1" Units="percent" Index="2" ViewPoint="Inside" Continuous="true">
+        <Sound WwiseEvent="ISpoolHighREXT" WwiseData="true" NodeName="Engine_PIVOT_RIGHT_2" SimVar="TURB ENG N1" Units="percent" Index="2" ViewPoint="Outside" Continuous="true">
             <Range LowerBound="1" />
             <WwiseRTPC SimVar="TURB ENG N1" Units="percent" Index="2" RTPCName="SIMVAR_TURB_ENG_N1" />
         </Sound>
 
-        <Sound WwiseEvent="ISpoolHighLEXT" WwiseData="true" NodeName="Engine_PIVOT_LEFT_2" SimVar="TURB ENG N1" Units="percent" Index="1" ViewPoint="Outside" Continuous="true">
+
+
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:245]:
```suggestion
```

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml [hunk 3]
```diff
@@ -277,26 +303,27 @@
...
             <WwiseRTPC SimVar="TURB ENG N1" Units="percent" Index="1" RTPCName="SIMVAR_TURB_ENG_N1" />
         </Sound>
 
-        <Sound WwiseEvent="RumbleLowR" NodeName="Engine_PIVOT_RIGHT_2" WwiseData="true" SimVar="TURB ENG N1" Units="percent" Index="2" Continuous="true">
+        <Sound WwiseEvent="ISpoolREXT" WwiseData="true" NodeName="Engine_PIVOT_RIGHT_2" SimVar="TURB ENG N1" Units="percent" Index="2" ViewPoint="Outside" Continuous="true">
             <Range LowerBound="1" />
             <WwiseRTPC SimVar="TURB ENG N1" Units="percent" Index="2" RTPCName="SIMVAR_TURB_ENG_N1" />
         </Sound>
 
+
```
Comment (Benjozork) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml:326]:
```suggestion
```

**Comment by hotshotp** [comment]
apparently gear is not making a sound, i will figure that out

**Comment by wpine215** [comment]
@hotshotp moving to draft until gear issue is resolved, to prevent any accidental merges in the meantime

**Comment by Sadawalebg** [comment]
And rework sound cruise ?

**Comment by hotshotp** [comment]
> And rework sound cruise ? 

I already responded to someone else to this question, that is a lot more difficult and will be added in another PR

**Comment by hotshotp** [comment]
Alright, I have made it so the gear sound plays now, just need someone to test it now but it should be good now

**Comment by kberg93** [comment]
Category: QA Tester
Name: Kberg#0666
Date of testing: Current date 11/27/2020
Version of the sim: 11.6.0
PR Tested: #2106
Object of test: Clunk sound on gears up and down

Testing technique: Entered the sim to test the PR
Steps you've made to test the Pull Request:

1. C&D startup and took off, clunk sound works on both gear up and down 👍🏻 

Flight notes (If required):

Issues (If Any):

Overall Rating: Good

Conclusions of the testing: Diggin' it!

**Comment by MapleNAV-TCASFAULT** [comment]
Category: QA Tester
Name: Maple#3904
Date of testing: Current date 11/27/2020
Version of the sim: 1.11.6.0
PR Tested: #2106 
Object of test: Aircraft sounds (specifically the gear)

Testing technique: Entered the sim to test the PR
Steps you've made to test the Pull Request:
1. Spawned at CYYZ runway 23, raised the gear, heard it
2. Slewed to a 5 mile approach position, lowered the gear and landed, heard it
3. 
4.
5.

Flight notes (If required):

Issues (If Any): 
Overall Rating: Good
Conclusions of the testing: Gave my full review in the previous report. Gear sound is now audible, plz merge!

**Comment by biozhard** [comment]
Category: QA Tester
Name: Peixeinfinit
Date of testing: Current date 11/27/2020
Version of the sim: 1.11.6.0
PR Tested: #2106
Object of test: GPS accuracy

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:

1. Spawned Cold & Dark at a gate
2. Performed normal operations to takeoff
3. Did a fly SBRJ to SBSP
4. When I activate the AP, the airplane turns just right and down. After the TA, he keep’s heading 30°, not following the route, and heading commands.
If I restart the ADIR’s in Flight through the a320x Options Alling instant, he follow the route, but with an inaccuracy of 10°

Issues (If Any): GPS Accuracy
Overall Rating: bad
Conclusions of the testing: Since of USA World Update this trouble started.

**Comment by kberg93** [comment]
> Category: QA Tester
> Name: Peixeinfinit
> Date of testing: Current date 11/27/2020
> Version of the sim: 1.11.6.0
> PR Tested: #2106
> Object of test: GPS accuracy
> 
> Testing technique: Description of how you tested the Pull Request
> Steps you've made to test the Pull Request:
> 
> 1. Spawned Cold & Dark at a gate
> 2. Performed normal operations to takeoff
> 3. Did a fly SBRJ to SBSP
> 4. When I activate the AP, the airplane turns just right and down. After the TA, he keep’s heading 30°, not following the route, and heading commands.
>    If I restart the ADIR’s in Flight through the a320x Options Alling instant, he follow the route, but with an inaccuracy of 10°
> 
> Issues (If Any): GPS Accuracy
> Overall Rating: bad
> Conclusions of the testing: Since of USA World Update this trouble started.

This is for the Sounds not GPS Accuracy :)

**Comment by hotshotp** [comment]
> Category: QA Tester
> Name: Peixeinfinit
> Date of testing: Current date 11/27/2020
> Version of the sim: 1.11.6.0
> PR Tested: #2106
> Object of test: GPS accuracy
> 
> Testing technique: Description of how you tested the Pull Request
> Steps you've made to test the Pull Request:
> 
> 1. Spawned Cold & Dark at a gate
> 2. Performed normal operations to takeoff
> 3. Did a fly SBRJ to SBSP
> 4. When I activate the AP, the airplane turns just right and down. After the TA, he keep’s heading 30°, not following the route, and heading commands.
> If I restart the ADIR’s in Flight through the a320x Options Alling instant, he follow the route, but with an inaccuracy of 10°
> 
> Issues (If Any): GPS Accuracy
> Overall Rating: bad
> Conclusions of the testing: Since of USA World Update this trouble started.
> 

Wrong pr bud

**Comment by Tibu73** [comment]
Great work with sounds but I miss the PTU sound after engine starts. I Know is not very loud in the cockpit but it's hearing too.

**Comment by hotshotp** [comment]
> Great work with new sounds but I miss the PTU sound after engine starts. I Know is not very loud in the cockpit but it's hearing too.

i answered you in another comment you made about the same issue, the PTU is NOT AUDIBLE in the cockpit of an a320 neo. The a320neo has insulation which completely prevents it from being heard in the cockpit. the only aircraft you will hear it on is the a318 because its much shorter

**Comment by richardkennedy** [comment]
> > Great work with new sounds but I miss the PTU sound after engine starts. I Know is not very loud in the cockpit but it's hearing too.
> 
> i answered you in another comment you made about the same issue, the PTU is NOT AUDIBLE in the cockpit of an a320 neo. The a320neo has insulation which completely prevents it from being heard in the cockpit. the only aircraft you will hear it on is the a318 because its much shorter

Whilst it may not be authentic is there a way to restore it, via say a user preferences file? For some reason it added a significant amount of 'realism' from a personal experience. Appreciate that it goes against the purpose of the project, so happy to accept a straight 'No'.

**Comment by Benjozork** [comment]
You are free to restore the sounds as you wish as this is an open-source project. However, I am afraid we will not provide that particular sound option.

**Comment by hotshotp** [comment]
> > > Great work with new sounds but I miss the PTU sound after engine starts. I Know is not very loud in the cockpit but it's hearing too.
> > 
> > i answered you in another comment you made about the same issue, the PTU is NOT AUDIBLE in the cockpit of an a320 neo. The a320neo has insulation which completely prevents it from being heard in the cockpit. the only aircraft you will hear it on is the a318 because its much shorter
> 
> Whilst it may not be authentic is there a way to restore it, via say a user preferences file? I found that it added a significant amount of additional 'realism' from a personal perspective, but I appreciate that it goes against the purpose of the project - so happy to accept a straight 'No'.

Soon you will be able to open the cockpit door. Once you can do that, it will definitely be audible in the cockpit, you can look forward to that

**Pull request merged**